Makefile

```diff
@@ -102,7 +102,7 @@ run-operator: generate fmt vet
 # Install CRDs into a cluster
 .PHONY: install
 install: crd
-	oc apply -f config/crds
+	kubectl apply -f config/crds
```

Comment:
On first blush, I do not think that this is a change that we want to make. We are deliberately specifying a dependency on `oc` rather than on `kubectl`. Can you give more color as to what `kubectl` is providing over `oc` here?

Reply:
Use of `kubectl` is just to stay more focused on upstream products, all that I've been workng with works with kubectl to be more compliant with Kubernetes itself, is not regarding the funcionality but compatibility.